commands.js

```diff
@@ -73,39 +73,62 @@ async function loop_check_bulk_transfer(scan_request_id) {
     return new Promise((resolve, reject) => {
         const intervalId = setInterval(async () => {
             try {
-                const info = await get_scan_request({ scan_request_id });
-                console.log(`Scan request with id: ${scan_request_id} has status: ${info.status}`);
-                
+                const info = await get_scan_request({ scan_request_id })
+                console.log(`Scan request with id: ${scan_request_id} has status: ${info.status}`)
```

Comment:
can we add some type of standardized Linting so that we don't get these types of diffs

Reply:
We can enable or disable it. What do you prefer? 
![image](https://github.com/deezy-inc/sat-hunter/assets/126987350/52293e74-1152-4566-9b3d-c44d1df9e9e5)